src/0_utils.js

```diff
@@ -8,7 +8,7 @@ goog.provide('utils');
 var DEBUG = true;
 
 /** @typedef {string} */
-message;
+var message;
```

Comment:
Ah, I think this should actually stay as-is - the idea here is to define a type to the closure compiler, not to actually initialize a variable